WalletWasabi.Gui/Controls/WalletExplorer/WalletViewModel.cs

```diff
@@ -93,23 +94,26 @@ public void OnWalletOpened()
 		{
 			Disposables = Disposables is null ? new CompositeDisposable() : throw new NotSupportedException($"Cannot open {GetType().Name} before closing it.");
 
-			var observed = Observable.Merge(
-				Global.UiConfig.WhenAnyValue(x => x.LurkingWifeMode).Select(_ => Unit.Default),
+			Observable.Merge(
 				Observable.FromEventPattern(Global.WalletService.TransactionProcessor, nameof(Global.WalletService.TransactionProcessor.CoinReceived)).Select(_ => Unit.Default),
 				Observable.FromEventPattern(Global.WalletService.TransactionProcessor, nameof(Global.WalletService.TransactionProcessor.CoinSpent)).Select(_ => Unit.Default))
-				.Throttle(TimeSpan.FromSeconds(1))
-				.ObserveOn(RxApp.MainThreadScheduler);
-
-			observed.Subscribe(_ =>
+				.Throttle(TimeSpan.FromSeconds(0.5))
+				.Merge(Global.UiConfig.WhenAnyValue(x => x.LurkingWifeMode).Select(_ => Unit.Default))
```

Comment:
Why was this needed here?

Reply:
It was there before, just moved down it is no need to be throttled.

Reply:
When you click on the balance to switch LWM on/off there is a delay that makes the application looks slow. @molnard suggested this change and now it works nicer.